Update syntax-for-issue-forms.md (#25277)
github/docs: content/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms.md

| `description` | A description for the issue form template, which appears in the template chooser interface. | Required | String |
| `body` | Definition of the input types in the form. | Required | Array |
| `assignees` | People who will be automatically assigned to issues created with this template. | Optional | Array or comma-delimited string |

> | `labels` | Labels that will automatically be added to issues created with this template. Be aware, that if a label does not already exist on the repository, it will not be automatically suggested on the resulting issue created from the template. | Optional | Array or comma-delimited string |

Reviewer: ```suggestion
| `labels` | Labels that will automatically be added to issues created with this template. If a label does not already exist in the repository, it will not be automatically added to the issue. | Optional | Array or comma-delimited string |
```
